docset/windows/appx/Get-NonRemovableAppsPolicy.md

```diff
@@ -0,0 +1,169 @@
+
+|||
+|-|-|
+| Type: | SwitchParamter |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -Path
+Specifies the full path to the root directory of the offline Windows image that you will inquire into. If the directory named Windows is not a subdirectory of the root directory, WindowsDirectory must be specified.
+
+|||
+|-|-|
+| Type: | String |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -ScratchDirectory
+Specifies a temporary directory that will be used when extracting files for use during inquiry. The directory must exist locally. If not spcified, the ``` \Windows\%Temp% ``` directory will be used, with a subdirectoryname of a randomly generated hexadecimal value for each run of DISM. Items in the scratch directory are deleted after each operation. You should not use a network share location as a scratch directory to expand a pachage(.cab or .msu file) for installation. The directory used for extracting files for temporary usage during servicing should be a local directory.
+
+|||
+|-|-|
+| Type: | String |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -SystemDrive
+Specifies the path to the location of the BootMgr files. This is necessary only when the BootMgr files are located on a partition other than the one that you are running the command from. Use -SystemDrive to inquire an installed Windows image from a Windows PE environment.
+
+|||
+|-|-|
+| Type: | String |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -WindowsDirectory
+Specifies the path to the Windows directory relative to the image path. This cannot be the full path to the Windows directory; it should be a relative path. If not specified, the default is the Windows directory in the root of the offline image directory.
+
+|||
+|-|-|
+| Type: | String |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -LogPath
+Specifies the full path and file name to log to. If not set, the default is ``` %WINDIR%\Logs\Dism\dism.log ```. In Windows PE, the default directory is the RAMDISK scratch space which can be as low as 32MB. The log file will automatically be archived. The archived log file will be saved with .bak appended to the file name and a new log file will be generated. Each time the log file is archived the .bak file will be overwritten. When using a network share that is not joined to a domain, use the net use command together with domain credentials to set access permissions before you set the log path for the DISM log.
```

Comment:
Use single backticks for in-line code. Instead of 'low as', use 'small as'.

Reply:
Updated.